src/nodes/tplink_status.ts

```diff
@@ -90,13 +102,23 @@ const nodeInit: NodeInitializer = (RED): void => {
                     result: false
                 };
 
+                let retEnergy: tplinkTapoConnectWrapperType.tapoConnectResults = {
+                    result: false
+                };
+
                 if (checkParameter(config)) {
                     // node: status
                     ret = await getTapoDeviceInfo(config);
+                    if (ret.tapoDeviceInfo?.model === "P110") {
+                        retEnergy = await getTapoEnergyUsage(config);
```

Comment:
Thank you for cooperating with my project.

It may be better to include getTapoEnergyUsage () in gettapodeviceinfo ().
This is the purpose of making one return value.

``
msg.payload = ret;
msg.payload.energy = retEnergy.tapoDeviceInfo;
``